src/omero_zarr/util.py

```diff
@@ -33,3 +35,69 @@ def open_store(name: str) -> FSStore:
         normalize_keys=False,
         mode="w",
     )
+
+
+def marshal_axes(
+    image: ImageWrapper, levels: int = 1, multiscales_zoom: float = 2.0
+) -> Dict[str, List]:
```

Comment:
:+1: but this is doing more than just marshalling the axes no?

Reply:
Yes, but naming things is hard. I didn't think `marshal_axes_and_transformations()` was so neat.
Any suggestions?

Reply:
Since `levels` and `multiscales_zoom` are irrelevant for the transformations marshalling, a potential option would be to splitting this into 2 separate calls?